acquia_cms.install

```diff
@@ -26,16 +26,6 @@ function acquia_cms_install() {
   }
 }
 
-/**
```

Comment:
@chandan-singh7929 I believe this needs to be updated for acquia_cms 1.5.x and 2.0.x release as well ??

Reply:
I don't think so, removing this because of new package manager.